akka-docs/src/main/paradox/stream/operators/FileIO/fromFile.md

```diff
@@ -10,13 +10,9 @@ The `fromFile` operator has been deprecated, use @ref:[fromPath](./fromPath.md)
 
 @@@
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [FileIO.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/FileIO.scala) { #fromFile }
-
-@@@
+@apidoc[fromFile](FileIO$) { scala="#fromFile(f:java.io.File,chunkSize:Int):akka.stream.scaladsl.Source[akka.util.ByteString,scala.concurrent.Future[akka.stream.IOResult]]" java="#fromFile(java.io.File)" }
```

Comment:
It would make sense to show the class name in the link text.
`@apidoc[FileIO.fromFile](FileIO$)`

Reply:
Yeah, good idea.